### DISCUSSION TITLE
Join

### DISCUSSION THREAD

**Discussion by xItzDiamondx** [ScratchAddons/ScratchAddons#2093] [Help] [answered]
Hey. So, I would like to join you in developing this. It is possible?

**Comment by retronbv** [comment, marked as answer]
Anyone can join, read the instructions on the wiki!

**Reply by retronbv** [reply]
Link to the wiki: https://github.com/ScratchAddons/ScratchAddons/wiki/

**Reply by xItzDiamondx** [reply]
Which section do I click on?

**Reply by xItzDiamondx** [reply]
Also, I would like to work on the web development, or something that is open.

**Reply by retronbv** [reply]
> Also, I would like to work on the web development, or something that is open.

Join the discord

**Reply by xItzDiamondx** [reply]
Is that a must or optional?

**Reply by cobaltt7** [reply]
we can help you understand things, join the discord so we don't spam this

https://discord.gg/Ak8sCDQ

**Reply by xItzDiamondx** [reply]
K

**Comment by WorldLanguages** [comment]
Join the Discord server, link in README

**Reply by xItzDiamondx** [reply]
Question for you on your scratch profile, or I will ask you here.

**Reply by WorldLanguages** [reply]
Ok no worries, but if you can it's recommended you participate on the discord server

**Comment by cobaltt7** [comment]
Yeah no need to ask
________________________________
From: xDiamondx ***@***.***>
Sent: Tuesday, April 6, 2021 2:51:44 PM
To: ScratchAddons/ScratchAddons ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ScratchAddons/ScratchAddons] Join (#2093)


Hey. So, I would like to join you in developing this. It is possible?

**Comment by cobaltt7** [comment]
Click on code and scroll down
